server/src/mds3.gdc.js

```diff
@@ -968,7 +970,7 @@ async function snvindel_byisoform(opts, ds) {
 
 	const [re_ssms, re_cases] = await Promise.all([p1, p2])
 
-	if (serverconfig.debugmode) console.log('gdc snvindel tandem queries', Date.now() - starttime)
+	mayLog('gdc snvindel tandem queries', Date.now() - starttime)
```

Comment:
Is this how to check the time it takes to load? Barchart seems very slow in the test links provided.

Reply:
it reports amount of time taken for api response. on my local it's 30+ seconds for the example without filter